app/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="quest_sport_manySports_confirmation_title">"You selected many sports"</string>
     <string name="quest_busStopShelter_name_title">"Does the bus stop \"%s\" have a shelter?"</string>
 	<string name="quest_toiletsFee_title">"Do these toilets require a fee?"</string>
+    <string name="quest_baby_changing_table_title">"Does this location have a baby changing table?"</string>
```

Comment:
It might make sense to have two different titles here:

- Does this **toilet** have a baby changing table?
- Does "Future Mall" have a baby changing table?

(only ask for diaper changing table if the non-toilet location has a name. This makes sense anyway so that people know which location is meant)